README.md

```diff
@@ -152,11 +142,47 @@ INFO: seed corpus: files: 755 min: 1b max: 35982b total: 252654b rss: 35Mb
 
 The crash is saved under `/tmp/fuzzing/artifacts` and can be further inspected.
 
-### Defining a Java fuzz test
+### Java fuzzing
+
+You can write `java_fuzz_tests` through the [Jazzer][jazzer-doc] fuzzing engine. You will need to enable it in your WORKSPACE `rules_fuzzing_dependencies` call:
+
+```python
+rules_fuzzing_dependencies(jazzer = True)
+
+load("@rules_fuzzing//fuzzing:init.bzl", "rules_fuzzing_init")
+
+rules_fuzzing_init()
+
+load("@jazzer//:repositories.bzl", "jazzer_dependencies")
+
+jazzer_dependencies()
+
+load("@jazzer//:init.bzl", "jazzer_init")
+
+jazzer_init()
+```
+
+To use Jazzer, it is convienient to also define a `.bazelrc` configuration, similar to the C++ libFuzzer one above:
+
+```
+# Force the use of Clang for all builds (needed to build Jazzer).
+build --action_env=CC=clang-10
+build --action_env=CXX=clang++-10
```

Comment:
Fixing a particular version of `clang` may be problematic here (although it is of course the right thing to do in `rules_fuzzing`'s own `.bazelrc`). How about:

```suggestion
# Force the use of Clang for all builds (Jazzer requires at least clang 9).
build --action_env=CC=clang
build --action_env=CXX=clang++
```

Reply:
Good catch - fixed.